src/components/orders/AmountsDisplay/styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+
+import TokenImg from 'components/common/TokenImg'
+
+export const Wrapper = styled.div`
+  display: grid;
+  row-gap: 1rem;
+  justify-items: start;
+  align-items: center;
+  grid-template-columns: 10rem auto;
+  padding: 1.3rem 0;
+`
+
+export const RowTitle = styled.span`
+  display: flex;
+  align-items: center;
+  font-weight: ${({ theme }): string => theme.fontBold};
+
+  &::before {
+    content: '▶';
+    margin-right: 0.5rem;
+    color: ${({ theme }): string => theme.icon};
+    font-size: 0.75rem;
+  }
+`
+
+export const RowContents = styled.span`
+  display: flex;
+  align-items: center;
+
+  & > span:first-child {
```

Comment:
@alfetopito Made it a tiny bit more specific by adding the `span` just before :first-child

Reply:
I addressed your comments here https://github.com/gnosis/gp-ui/pull/209
One thing to notice is that by removing the wildcard rule that was adding margin-left, the elements are now glued together
See [this PR](https://pr208--gpui.review.gnosisdev.com/rinkeby/orders/0xa6223509c0a41f992995d8781c3452ae8e2f55640b21d057fc02c7fb5a1674b35b0abe214ab7875562adee331deff0fe1912fe4260246928/) vs [my changes](https://pr209--gpui.review.gnosisdev.com/rinkeby/orders/0xa6223509c0a41f992995d8781c3452ae8e2f55640b21d057fc02c7fb5a1674b35b0abe214ab7875562adee331deff0fe1912fe4260246928/)
![screenshot_2021-02-25_10-27-23](https://user-images.githubusercontent.com/43217/109199227-07fe5a00-7754-11eb-9f49-616689e27dd7.png)
![screenshot_2021-02-25_10-28-03](https://user-images.githubusercontent.com/43217/109199258-12b8ef00-7754-11eb-8d2d-3a56644e41c8.png)

Reply:
Merged yours in and tested, all good now :)